inorbit_republisher/README.md

```diff
@@ -119,6 +119,13 @@ The `mapping_options` for this type include:
   data: "linear_vel={\"y\": 0.00013548378774430603, \"x\": 0.0732172280550003, \"z\": 0.0}"
   ```
 
+  * `filter`: a lambda expression that can be used to control whether or not the JSON object is published based on a condition. For example, if you'd like to republish only JSON objects that have a value for the key ``important_key`` different than ``NOT IMPORTANT``, you can do it with:
```

Comment:
I suggest using more familiar and realistic examples. Such as the one in the previous example - you might want to use the same one and filter for `z!=0` or something like that